Fix/ Risk of duplicate range formats when applying format from another datapoint #2769

Patrykb0802 opened the pull request:
Issue: #2769 

Added:
- getRangeValues() method which returns all current range text renderers
- Validation for text renderers when using "Apply new properties basing on", now there should be no duplicates

Patrykb0802 committed:
Risk of duplicate range formats when applying format from another datapoint #2769

Added:
- getRangeValues() method which returns all current range text renderers
- Validation for text renderers when using "Apply new properties basing on", now there should be no duplicates

Limraj commented on WebContent/WEB-INF/jsp/dataPointEdit.jsp:
```diff
@@ -319,11 +319,20 @@
                                     console.log(message);
                                 }
                                 for (var range in properties.textRenderer.rangeValues) {
+                                    for (let i = 0; i < thisRangeValues.length; i++) {
```
This needs to be corrected similarly for the 'Event Text Renderer' type

Patrykb0802 committed:
Fix/ Risk of duplicate range formats when applying format from another datapoint #2769

- Added similar duplicate removal system for event text renderers as for text renderers
